src/Maker/MakeCrud.php

```diff
@@ -179,7 +179,8 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
                     'entity_var_singular' => $entityVarSingular,
                     'entity_twig_var_singular' => $entityTwigVarSingular,
                     'entity_identifier' => $entityDoctrineDetails->getIdentifier(),
-                    'use_render_form' => method_exists(AbstractController::class, 'renderForm'),
+                    // @legacy - Remove when support for Symfony <6 is dropped - hack to determine version
+                    'use_render_form' => method_exists(AbstractController::class, 'getDoctrine'),
```

Comment:
Please use `Kernel::VERSION` to determine version

Reply:
Agreed. We normally like to check directly for the feature we're using. But in this case, as you @nacorp mentioned, that's not possible. So let's just check the kernel version :)

Reply:
Here we go !